h2o-dist/index.html

```diff
@@ -493,152 +512,161 @@ <h1><span>H<sub>2</sub>O</span></h1>
+                    if ("h2o" %in% rownames(installed.packages())) { remove.packages("h2o") }<br>
+                    <br>
+                    <em># Next, we download packages that H2O depends on.</em><br>
+                    pkgs &lt;- c("RCurl","jsonlite")<br>
+                    for (pkg in pkgs) {<br>
+                      if (! (pkg %in% rownames(installed.packages()))) { install.packages(pkg) }<br>
+                    }<br>
+                    <br>
+                    <em># Now we download, install and initialize the H2O package for R.</em><br>
+                    install.packages("h2o", type="source", repos="<script type="text/javascript">var url = location.href; var url2 = url.substring(0, url.lastIndexOf("/")); document.write(url2);</script>http://h2o-release.s3.amazonaws.com/h2o/rel-xia/2/R")<br>
+                    <br>
+                    <em># Finally, let's load H2O and start up an H2O cluster</em><br>
+                    library(h2o)<br>
+                    h2o.init()<br>
+            </p></div>
 
           <div id="quickstart-hadoop" style="display:none">
-            <h2>Run H<sub>2</sub>O on Hadoop in just 3 steps</h2>
-            <p>1. Download H<sub>2</sub>O for your version of Hadoop. This is a zip file that contains everything you need to get started.</p>
-            <button class="btn copy_button" id="btnCopy4" data-copied-hint="Copied!" data-clipboard-target='to_copy4'>
+            <h2  class="get-started firststep">Run H2O on Hadoop in just 3 steps</h2>
+            <p  class="get-started-p">1. Download H2O for your version of Hadoop. This is a zip file that contains everything you need to get started.</p>
+            <button class="btn copy_button" id="btnCopy4" data-copied-hint="Copied!" data-clipboard-target="to_copy4">
               <span class="octicon octicon-clippy"></span>
             </button>
-            <p class="terminal" id="to_copy4"></p>
-            <p>2. Unpack the zip file and launch a 6g instance of H<sub>2</sub>O: </p>
-             <button class="btn copy_button" id="btnCopy7" data-copied-hint="Copied!" data-clipboard-target='to_copy7'>
+            <p class="terminal" id="to_copy4"><ul></ul></p>
+            <p  class="get-started-p">2. Unpack the zip file and launch a 6g instance of H2O: </p>
+             <button class="btn copy_button" id="btnCopy7" data-copied-hint="Copied!" data-clipboard-target="to_copy7">
               <span class="octicon octicon-clippy"></span>
             </button>
             <p class="terminal" id="to_copy7">
-                unzip h2o-SUBST_PROJECT_VERSION-*.zip<br/>
-                cd h2o-SUBST_PROJECT_VERSION-*<br/>
-                hadoop jar h2odriver.jar -nodes 1 -mapperXmx 6g<br/>
+                unzip h2o-SUBST_PROJECT_VERSION-*.zip<br>
+                cd h2o-SUBST_PROJECT_VERSION-*<br>
+                hadoop jar h2odriver.jar -nodes 1 -mapperXmx 6g -output hdfsOutputDirName<br>
```

Comment:
Note the -output flag was removed.

Reply:
Can you please clarify if the output flag is supposed to be removed?